scalableminds/webknossos
# Encode layer visibility in sharing link

### URL of deployed dev instance (used for testing):
- https://sharelayervisibility.webknossos.xyz

### Steps to test:
- Open an annotation, enable and disable layers, set the annotation to public and copy the sharing link
  - Open the annotation in an incognito tab, the layer visibility should be equal to the original annotation (also for the Skeleton layer)
  - Reload the original annotation, the layer visibility should remain the same

### Issues:
- contributes to #5080

------
(Please delete unneeded items, merge only when none are left open)
